worker/sshserver/package_test.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2025 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package sshserver_test
+
+import (
+	stdtesting "testing"
+
+	"github.com/juju/juju/testing"
+)
+
+func TestPackage(t *stdtesting.T) {
+	testing.MgoTestPackage(t)
```

Comment:
How do you know whether you need the Mongo testing package versus the other testing packages?

Reply:
I don't know, the only other I saw was SSL?